python-package/xgboost/sklearn.py

```diff
@@ -574,9 +548,6 @@ def load_model(self, fname):
             if k == 'classes_':
                 self.classes_ = np.array(v)
                 continue
-            if k == 'use_label_encoder':
-                self.use_label_encoder = bool(v)
-                continue
```

Comment:
Why is this line being removed?

Reply:
We reuse the doc from `booster.load_model`.  See line 563

Reply:
bool value is handled by JSON.

Reply:
Given the test, I take your word here.